datajoint/table.py

```diff
@@ -486,6 +486,7 @@ def delete(
         transaction: bool = True,
         safemode: Union[bool, None] = None,
         force_parts: bool = False,
+        include_parts: bool = True,
```

Comment:
I'd suggest a different argument name. It's not super obvious what `include_parts` means, as ones must expect that cascade delete will also delete the parts.

Here, we mean to include also the "master" table(s) of the part table(s) being deleted.

Suggesting to rename to something like `force_masters` (for consistency with the arg `force_parts`)
Or perhaps consider using an environment variable or configuration (in `dj.config`) to control for this behavior (similar to `DJ_SUPPORT_FILEPATH_MANAGEMENT`)

Reply:
I would prefer to have a single consistent behavior, not making it configurable.